system/security/playlist.xml

```diff
@@ -41,6 +41,10 @@
 		<levels>
 			<level>extended</level>
 		</levels>
+		<versions>
+			<version>8</version>
+			<version>11</version>
+		</versions>
```

Comment:
Is this a temporary exclusion? If so, please use disable feature and reference the open issue. Thanks.

Reply:
Quick answer is we don't know.  @joeyleeeeeee97 could probably confirm that https://github.com/adoptium/aqa-systemtest/issues/474

Reply:
@sophia-guo could we use disable feature and reference adoptium/aqa-systemtest#474 for now?

Reply:
that is a good approach as this is a temporary exclusion based on joeyleeeeeee97's reply